### ISSUE TITLE
9292ov.nl does not work (any more) in Belgium

### ISSUE THREAD

**Issue by M1dgard** [smurfy/fahrplan#240]
I can't find any location in Belgium from within the app. I tried on [9292ov.nl](http://9292ov.nl) and it gave the same results. Apparently they don't do Belgium (any more). So Belgium should be removed from the description.

Would be great to include one of the following APIs for Belgium:

* https://hello.irail.be/api/
    * Flanders and Brussels
        * realtime train
        * static bus (i.e. only times according to timetables, not taking into account delays etc.)
    * Wallonia
        * realtime train

* De Lijn API
    * Flanders
        * realtime bus
        * realtime train
    * Wallonia and Brussels
        * realtime train

De Lijn is the Flemish bus company. Their API is documented in Dutch at http://docs.delijn.apiary.io/.

**Comment by smurfy** [comment]
just tested on sailfish. workes for me. what version and platform are you using?

**Comment by M1dgard** [comment]
2.0.26 on Ubuntu Touch with OTA-10.1, but I actually think 9292ov.nl has a problem. I did some more experiments with Fahrplan and their own site, both always give the same results:

Searching for "Gent-Dampoort" brings up bus stop *Dampoort* in Ghent instead of the train station *Gent-Dampoort*.

"Brugge", "Knokke" etc. return the place, but not the train station.

For some stations it shows streets in the Netherlands: "Blankenberge" gives the two *Blankebergestraat*s in Breda and Amsterdam.

But then again, some stations are found: "Brussel Centraal", "Brussel Zuid", "Brussel Noord", "Hasselt", "Sint-Niklaas".

Those are some *very* inconsistent results.

**Comment by leppa** [comment]
If Fahrplan returns exactly the same results as 9292ov.nl website, then it's nothing we can fix. We just display whatever the backend service (being 9292ov.nl in this case) returns to us.

However, if there are differences in what you see on the website and in the app, then let us know (like #241 that you reported). We'll try to fix it.

As for the API you suggested, it should be possible to implement it. However neither I, nor @smurfy seem to have enough spare time at the moment. So any contributions are appreciated.

**Comment by M1dgard** [comment]
> If Fahrplan returns exactly the same results as 9292ov.nl website, then it's
> nothing we can fix. We just display whatever the backend service (being
> 9292ov.nl in this case) returns to us.

Yep, that's why I was initially just asking to remove "Belgium" from the 
description. ;)

> As for the API you suggested, it should be possible to implement it. However
> neither I, nor @smurfy seem to have enough spare time at the moment. So any
> contributions are appreciated.

Who knows, if I have some time the coming weeks. :)

Thanks for the reply!

**Comment by d9h02f** [comment]
Either of those APIs would certainly be appreciated!
